### PR TITLE
Feat/add books languages

### PR DESCRIPTION
**Pull request by sagar-wednesday**
### Ticket Link

---

### Related Links

---

### Description
added books_languages table to establish many-to-many relationships between books and languages
---

### Steps to Reproduce / Test

---

---

### Checklist

- [ ] PR description included
- [ ] `yarn test` passes
- [ ] Tests are [changed or added]
- [ ] Relevant documentation is changed or added (and PR referenced)

### GIF's

---

### CODE REVIEW

File: resources/v9/01_books_languages.sql [hunk 1]
```diff
@@ -0,0 +1,14 @@
+CREATE TABLE books_languages (
+  id SERIAL PRIMARY KEY,
+  language_id integer NOT NULL,
+  book_id integer NOT NULL,
+  created_at timestamp 
+    WITH time zone DEFAULT NOW
+    (),
+    updated_at timestamp
```
Comment (alichherawalla) [inline resources/v9/01_books_languages.sql:8]: we need a trigger here to update this right?

File: server/database/models/languages.js [hunk 2]
```diff
@@ -37,7 +37,12 @@ export function model(sequelize, DataTypes) {
   });
 
   languages.associate = function(models) {
-    languages.hasMany(models.books, {
+    // languages.hasMany(models.books, {
```
Comment (alichherawalla) [inline server/database/models/languages.js]: lets remove if not needed

File: resources/v9/01_books_languages.sql (hunk 1 above)
Comment (sagar-wednesday) [reply resources/v9/01_books_languages.sql:8]: I already added that in the next PR

Comment (sagar-wednesday) [reply resources/v9/01_books_languages.sql:8]: the PR #9

File: server/database/models/languages.js (hunk 2 above)
Comment (sagar-wednesday) [reply server/database/models/languages.js]: Yeah I removed that in the PR #10

**Review by alichherawalla — APPROVED** [review]

**Pull request merged**